soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3603,6 +3611,9 @@ void GenerateRandomizerImgui() {
     cvarSettings[RSK_KAK_GATE] = CVar_GetS32("gRandomizeKakarikoGate", 0);
     cvarSettings[RSK_DOOR_OF_TIME] = CVar_GetS32("gRandomizeDoorOfTime", 0);
     cvarSettings[RSK_ZORAS_FOUNTAIN] = CVar_GetS32("gRandomizeZorasFountain", 0);
+    //Starting Age is forced to child if forest setting is set to closed. (0 = Adult, 1 = Child)
+    cvarSettings[RSK_STARTING_AGE] = ((CVar_GetS32("gRandomizeForest", 0)) && 
+                                        (CVar_GetS32("gRandomizeStartingAge", 0)));
```

Comment:
This comment is reversed now that you swapped the ages around. Also, doesn't this also force the option to be child when forest is set to "Closed Deku" as well? Also I'm not entirely sure, but does this still resolve to 2 when Link's age is set to random?

Reply:
No, eventually 3D rando will resolve the starting age to either 0 or 1.